client/src/app/shared/models/agenda/agenda-item.ts

```diff
@@ -36,6 +36,7 @@ export class AgendaItem extends BaseModel<AgendaItem> {
     public duration: number; // in seconds
     public weight: number;
     public level: number;
+    public tree_weight: number;
```

Comment:
I don't know if this is the right place for this information.

Reply:
Ok, please add a comment, that is is now client calculated. Please also set the level in the client and mark `level` as client-calculated here, too.